docs/content/sidebars/concepts.js

```diff
@@ -122,6 +122,18 @@ const concepts = [
 			},
 			'concepts/cryptography/zklogin',
 			'concepts/cryptography/passkeys',
+			{
+				type: 'category',
+				label: 'Nautilus',
+				link: {
+					type: 'doc',
+					id: 'concepts/cryptography/nautilus',
+				},
+				items: [
+					'concepts/cryptography/nautilus/nautilus-design',
+					'concepts/cryptography/nautilus/using-nautilus',
```

Comment:
Are we married to this setup? I think the nautilus design and overview page should be combined and the using nautilus page should go in the Guides section. There should then be a Related links level 2 heading at the end of each topic that points to each other...and other links that might be valuable.

I can do this to see if you like or no. Let me know.

Reply:
Hmm.. I get you, but would be great if we could leave it as is for now, to allow the builders who've seen the Github docs find the same format (that's how we structured it there).. Basically we're just looking to migrate for mainnet launch right now.. We could then do your suggested optimization a couple of months later as an improvement once we've a few more things to add (like TEE network partners that we would've by then)..